lib/libs/handler-lib.ts

```diff
@@ -1,21 +1,3 @@
-import type { APIGatewayEvent, APIGatewayProxyResult, Context } from "aws-lambda";
-
-export const handler = async (
-  handler: (event?: APIGatewayEvent, context?: Context) => Promise<APIGatewayProxyResult>,
-) => {
-  const handlerResponse = await handler();
-
-  const response: APIGatewayProxyResult = {
-    headers: {
-      "Access-Control-Allow-Headers": "Content-Type",
-      "Access-Control-Allow-Origin": "*",
-      "Access-Control-Allow-Methods": "OPTIONS,POST,GET,PUT,DELETE",
-    },
-    ...handlerResponse,
-  };
-  return () => response;
-};
-
```

Comment:
I see you remove this. I'm guessing it's not used anywhere. Just wanted to ask to be sure?

Reply:
I didn't see it referenced anywhere in the code.